.github/workflows/maven-publish.yml

```diff
@@ -0,0 +1,34 @@
+# This workflow will build a package using Maven and then publish it to GitHub packages when a release is created
```

Comment:
Why did you add `maven-publish.yml `file ?

Reply:
i thought it was a file that can be helpful

Reply:
You should know what it does, before you add it

Reply:
And I don't think it's needed at this point